src/urllib3/util/ssltransport.py

```diff
@@ -193,9 +193,6 @@ def cipher(self) -> tuple[str, str, int] | None:
     def selected_alpn_protocol(self) -> str | None:
         return self.sslobj.selected_alpn_protocol()
 
-    def selected_npn_protocol(self) -> str | None:
-        return self.sslobj.selected_npn_protocol()
-
```

Comment:
You could argue that this is a breaking change, but we don't document SSLTransport so I think that's fine.